igb/repl.go

```diff
@@ -38,172 +38,150 @@ const (
 	emojis = "😀😁😂🤣😃😄😅😆😉😊😋😎😍😘😗😙😚🙂🤗🤔😐😑😶🙄😏😮😪😴😌😛😜😝🤤🙃🤑😲😭😳🤧😇🤠🤡🤥🤓😈👿👹👺💀👻👽🤖💩😺😸😹😻😼😽"
 )
 
+// Igb holds internal states of Igb.
+type Igb struct {
```

Comment:
Are you planning use this struct outside this Package? If not you shouldn't export it (and `Ivm`).